### PR TITLE
chore(scripts): add scripts

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

**Review by yinonov — COMMENTED** [review]
@YonatanKra objected this. lets hear is word on this

**Comment by YonatanKra** [comment]
I don't mind actually. If it helps most then we can go for it.

File: package.json [hunk 1]
```diff
@@ -3,9 +3,12 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "start": "nx serve",
-    "build": "nx build",
-    "test": "nx test",
+    "start": "npx nx run docs:serve",
+    "build": "npx nx build docs && npx nx build components",
```
Comment (YonatanKra) [inline package.json:7]: I think components should be built first

**Review by YonatanKra — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rinaok) [reply package.json]: @YonatanKra updated

Comment (yinonov) [reply package.json]: but why the redundancy? doesn't doc build components? it slows things down

Comment (YonatanKra) [reply package.json]: I think so. That's why these commands are redundant :)
I mean - what does this save you? Not much.
`npx nx build docs` => `npm run build`
Less declarative and and less `monorepo-ish` cause we have several builds. So what are you building? The components? the docs? the react demo app? the angular demo app?
The only thing I can agree upon is the `change` command which can be really annoying :)

Comment (rinaok) [reply package.json]: I don't mind about the build but `npm start` vs `npx nx run docs:serve` saves me a lot.

Comment (YonatanKra) [reply package.json]: @rinaok - ok, so let's remove the `build` command as it is redundant and leave `npm start` as an alias to the `docs:serve` command.
And another thing - if you are running commands from an `npm script` you don't need `npx` - just run `nx run docs:serve` and it'll work.
`npx` is when you are running from command line and unsure if you have the target library installed.

Comment (rinaok) [reply package.json]: @YonatanKra removed the build

**Review by YonatanKra — APPROVED** [review]

**Pull request merged**